src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,124 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.UserNotValidException;
+import core.basesyntax.model.User;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+class RegistrationServiceImplTest {
+    private static final String SHORTEST_VALID_LOGIN = "LoginN";
+    private static final String SHORTEST_VALID_PASSWORD = "123456";
+    private static final Integer MIN_VALID_AGE = RegistrationServiceImpl.MIN_AGE;
+    private RegistrationServiceImpl registrationServiceImpl;
+
+    @BeforeEach
+    void setUp() {
+        Storage.people.clear();
+        registrationServiceImpl = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void register_NullUser_NotOk() {
```

Comment:
fix the names of all test methods. Word before underscore (_) should start from lowercase 
```suggestion
    void register_nullUser_notOk() {
```